todo_app/templates/index.html

```diff
@@ -20,9 +20,31 @@ <h1 class="display-4">To-Do App</h1>
 <br>
   <div class="row">
     <div class="col-md-12">
-      <h2>Items</h2>
+      <h2>To Do:</h2>
       <ul class="list-group mb-4">
-        {% for items in my_items %}
+        {% for items in view_model.todo_items %}
+          <li> {{ items.name }} - {{ items.status }}</li>
+          {% if items.status != "Done"%}
+          <form action="complete-items/{{items.id}}" method="post">
+            <button>Complete</button>
+          </form>  
+          {% endif %}
+        {% endfor %}
+      </ul>
+      <h2>Doing:</h2>
+      <ul class="list-group mb-4">
+        {% for items in view_model.doing_items %}
+          <li> {{ items.name }} - {{ items.status }}</li>
+          {% if items.status != "Done"%}
+          <form action="complete-items/{{items.id}}" method="post">
+            <button>Complete</button>
+          </form>  
+          {% endif %}
+        {% endfor %}
+      </ul>
+      <h2>Done:</h2>
+      <ul class="list-group mb-4">
+        {% for items in view_model.done_items %}
           <li> {{ items.name }} - {{ items.status }}</li>
           {% if items.status != "Done"%}
```

Comment:
(Similarly, we can remove this and all the code inside this check, since all these items are "done" by definition!)